wp-content/themes/currentorg/partials/largo-header.php

```diff
@@ -1,21 +1,11 @@
 <header id="site-header" class="clearfix" itemscope itemtype="http://schema.org/Organization">
 	<?php largo_header(); ?>
 
-	<!-- <?php if (is_home()) { ?>
+	<?php ?>
 		<div class="newsletter-signup">
-			<form action="//current.us10.list-manage.com/subscribe/post?u=be91053a159d253f584056e47&id=1d564f03d8" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-				<label>Sign up for our newsletter</label>
-				<fieldset>
-					<input required type="email" value="" name="EMAIL" class="required email_address" id="mce-EMAIL" placeholder="Email address">
-					<input required type="text" value="" name="FNAME" class="required first_name" id="mce-FNAME" placeholder="First name">
-					<input required type="text" value="" name="LNAME" class="required last_name" id="mce-LNAME" placeholder="Last name">
-					<input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button submit"> -->
-					<!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-					<!-- <div style="position: absolute; left: -5000px;"><input type="text" name="b_be91053a159d253f584056e47_1d564f03d8" tabindex="-1" value=""></div>
-				</fieldset>
-			</form>
+			<a href="/subscribe/"><img src="/wp-content/themes/currentorg/img/subscribe.png"></a>
```

Comment:
This is an adequate way to link to the image, but a better way would be to use https://codex.wordpress.org/Function_Reference/get_stylesheet_directory_uri for something like:

```
<img src="<?php echo esc_attr( get_stylesheet_directory_uri() . '/img/subscribe.png' ); ?>">
```

[esc_attr](https://developer.wordpress.org/reference/functions/esc_attr/) is used here to make sure the URL is safe for output as an attribute.